solutions/cron/.env.example

```diff
@@ -1,4 +1,2 @@
-EDGE_CONFIG_ID=
-EDGE_CONFIG=
-TEAM_ID_VERCEL=
-ACCESS_TOKEN_VERCEL=
\ No newline at end of file
+UPSTASH_REDIS_REST_URL=
```

Comment:
i'm looking into this CRON project, and i'm quite confused what the upstash env variables are for - they don't look to be used anywhere? And looks to be complaining now for the KV env variables instead @steven-tey